### PR TITLE
Added swagger doc

### PR DESCRIPTION
**Pull request by KunalAnkur**
### Description
By fixing this issue I have used drf-yasg package in which we can only interact via swagger user interface. And removed all the error that @naveennvrgup sir have suggested.

**Type of Change:** 
- Code

**Code/Quality Assurance Only** 
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
You can test just by running python manage.py runserver after installing all the requirements.
till now there is no API in the app so we can just see the UI of swagger doc at http://localhost:8000/ url.

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged
- [ ] Update Swagger documentation and the exported file at /docs folder
- [x] Update requirements.txt

**Code/Quality Assurance**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: server/urls.py [hunk 1]
```diff
@@ -15,11 +15,31 @@
...
+from drf_yasg import openapi
+
+
+
+schema_view = get_schema_view(
+   openapi.Info(
+      title="teesco-API",
+      default_version='v1',
+      description="Test description",
+      terms_of_service="https://https://ecell.nitrr.ac.in/terms",
```
Comment (sumit-badsara) [inline server/urls.py:29]: remove one https://

**Review by sumit-badsara — CHANGES_REQUESTED** [review]
Good work @KunalAnkur ! Just a minor change.

**Comment by sumit-badsara** [comment]
Closed #41 , As this PR is more recent and updated.

**Comment by sumit-badsara** [comment]
@KunalAnkur Don't mention comments in PR title. 
Removed **Removed all suggested errors** from title.

**Comment by naveennvrgup** [comment]
1. Since no response from @KunalAnkur  for 4 days I am reassigning this issue to @sahil9001. 
1. @sahil9001  Please take the commits from this PR and resolve the conflicts
1. And address the comments made by @BadduCoder @naveennvrgup 
1. I am closing this PR

**Pull request closed without merging**